docs/metrics.md

```diff
@@ -1,4 +1,4 @@
-<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+<!-- AUTOGENERATED BY glean_parser. DO NOT EDIT. DO NOT COMMIT. -->
```

Comment:
I'd rather not, otherwise each time we make a change to the registry file we'll need to manually remove this. It will autofix itself once we move to a newer version of glean